docs/getting_started.md

```diff
@@ -328,6 +328,106 @@ See these resources for more information on automated cell type annotation:
 - [Assigning cell types with `SingleR`](https://bioconductor.org/books/release/SingleRBook/)
 - [Cell type assignment chapter in Orchestrating Single Cell Analysis](https://bioconductor.org/books/3.17/OSCA.advanced/cell-cycle-assignment.html)
 
+
+## Working with a merged ScPCA object
+
+Merged ScPCA objects contain all information found in `processed` objects for all individual libraries that compile a given ScPCA project.
+For more information on how these objects were prepared, see {ref}`the section on merged object preparation<processing_information:merged objects`.
+**Please be aware that data in merged objects has not been integrated/batch-corrected.**
+
+To work with a merged object, you will first have to read it in.
+
+You can read in a `SingleCellExperiment` merged object with the following R code:
+
+```r
+library(SingleCellExperiment)
+merged_sce <- readRDS("SCPCP000000_merged.rds")
+```
+
+You can read in an `AnnData` merged object with the following python code:
+
+```r
+import anndata
+adata_merged_object = anndata.read_h5ad("SCPCP000000_merged_rna.hdf5")
+```
+
+### Analyses with merged objects
+
+Merged object can facilitate joint analysis on multiple samples at the same time.
+Specifically, merged objects are useful for comparing _gene-level metrics_ among different samples.
+
+Some examples of analyses you can perform with a merged object include the following:
+
+- Differential expression analysis
+  - [Differential expression analysis chapter in Orchestrating Single Cell Analysis](https://bioconductor.org/books/3.17/OSCA.multisample/multi-sample-comparisons.html)
+  - [Differential gene expression analysis chapter in Single-cell Best Practices](https://www.sc-best-practices.org/conditions/differential_gene_expression.html)
+  - [10x resource: Differential gene expression analysis in scRNA-seq data between conditions with biological replicates](https://www.10xgenomics.com/resources/analysis-guides/differential-gene-expression-analysis-in-scrna-seq-data-between-conditions-with-biological-replicates)
+- Gene-set enrichment analysis
+  - [Gene set enrichment and pathway analysis chapter in Single-cell Best Practices](https://www.sc-best-practices.org/conditions/gsea_pathway.html)
+
+
+By contrast, analyses which compare _cell-level metrics_ among different samples require an **integrated/batch-corrected** object.
```

Comment:
Hmmm "require" might be a strong word. I think we should say that if you want to decrease batch effects then it's recommended to integrate? 
```suggestion
By contrast, it is recommended to control for batch effects when performing analyses that compare _cell-level metrics_ among different samples. 
In order to do this, a **integrated/batch-corrected** object should be created.
```